src/shared/model/experiments/abTests.ts

```diff
@@ -0,0 +1,41 @@
+import { AB, ABTest, Participations } from '@guardian/ab-core';
+import { switches } from './abSwitches';
+import { exampleTest } from './tests/example-test';
+
+interface ABTestConfiguration {
+  abTestSwitches: Record<string, boolean>;
+  arrayOfTestObjects: ABTest[];
+  pageIsSensitive: false;
+  mvtMaxValue: 1000000;
```

Comment:
As this mvtMaxVaule is used in multiple places, can it be defined as a `CONST` somewhere ?

Reply:
Maybe just in this file !

```
export const MVT_MAX_VALUE = 1000000

interface ABTestConfiguration {
  abTestSwitches: Record<string, boolean>;
  arrayOfTestObjects: ABTest[];
  pageIsSensitive: false;
  mvtMaxValue: MVT_MAX_VALUE;
 ...
```

Reply:
The interface itself already enforces this:
```ts
interface ABTestConfiguration {
  abTestSwitches: Record<string, boolean>;
  arrayOfTestObjects: ABTest[];
  pageIsSensitive: false;
  mvtMaxValue: 1000000;
}
```

so the mvtMaxValue cannot be set to any other value or else typescript will complain